packages/@react-spectrum/table/docs/TableView.mdx

```diff
@@ -1587,6 +1587,117 @@ function CustomDragPreviewExample() {
 <CustomDragPreviewExample />
 ```
 
+## Expandable Rows <VersionBadge version="alpha" style={{marginLeft: 4, verticalAlign: 'bottom'}} />
+
+By default, TableView doesn't allow expandable rows, but this can be enabled by toggling the feature flag via `enableTableNestedRows` in your app and then providing the `UNSTABLE_allowsExpandableRows` prop to
+your TableView.
+
+```tsx example
+import {enableTableNestedRows} from '@react-stately/flags';
```

Comment:
can we add where in your app this should go? eg the root vs the file that expandable rows is used? or anywhere? does it matter?

Reply:
I'll update this section with some clarification. Here is the entire explanation:

> This approach is the easiest because the flags can be accessed from anywhere. However, one downside is that flags are enabled for all instances of a component across the whole app, not for one specific table for example. We could do that with context, but then the flags would need to be wired to more places that aren't allowed to access classes (e.g. classes, non-hook functions, etc.). The flags also need to be set before anything is rendered in the app or things may break unexpectedly, so libraries like quarry should not be enabling flags, they should be enabled in apps. Like any other package, there must be only one copy installed or things will also not work.

From https://github.com/adobe/react-spectrum/pull/4617

Reply:
So you'll want to do this once, likely at the root of your app since it will affect all instances and should be done before rendering the table.